README.md

```diff
@@ -75,7 +75,37 @@ This command is used to setup the main command (/notify) by adding in the a user
 
 # Running The Bot
 
+This section outlines the steps required to get this project up and running on your local dev environment.
+
+## Prerequisites
+
+1. [Install Docker](https://docs.docker.com/get-docker/)
+2. [Install Docker Compose](https://docs.docker.com/compose/install/)
+
 ## Environment Variables
 
+For your local development environment setup, you will require a `.env` file. Create this file in the root directory of the project.
+Copy the contents of the `.env.example` file into this file, and populate the value for the `GUB_TESTING_BOT_TOKEN` key.
+
+### Generating a GUB_TESTING_BOT_TOKEN
+
+We will need an API token to access the [BotsAPI from telegram](https://core.telegram.org/bots/api). In order to generate one:
+
+1. Open the Telegram app using the desktop or web version
+2. In the search bar, type in `BotFather` and select the verified profile of BotFather
+3. Start the bot and issue the command `/newbot` in the chat to create a new testing bot
+4. Give your testing bot a name. Note: the bot name must end in `bot` (for example: TetrisBot or tetris_bot)
```

Comment:
I think you should replace this step and instead say something along the lines of "Once you have finished creating your testing bot, `@BotFather` will provide you with the link that you can use to access your newly created bot, which will eventually run the GUB bot, and `@BotFather` will also provide the token which you can insert into the .env file"

I worded it pretty roughly so you can clean it up, but essentially what I'm saying is we don't need to specify the steps to creating the bot itself. `@BotFather` will take care of that